packages/odyssey-storybook/src/components/odyssey-mui/Table/Table.stories.tsx

```diff
@@ -0,0 +1,472 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Story } from "@storybook/react";
+import * as React from "react";
+import Button from "@mui/material/Button";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import TableSortLabel from "@mui/material/TableSortLabel";
+import Checkbox from "@mui/material/Checkbox";
+import { MuiThemeDecorator } from "../../../../.storybook/components";
+import TableMdx from "./Table.mdx";
+
+export default {
+  title: `MUI Components/Table`,
+  component: Table,
+  parameters: {
+    docs: {
+      page: TableMdx,
+    },
+  },
+  argTypes: {
+    disabled: {
+      control: "boolean",
+      defaultValue: false,
+    },
+    icon: {
+      control: "text",
+      defaultValue: null,
+    },
+    label: {
+      control: "text",
+      defaultValue: "Asteroids",
+    },
+    wrapped: {
+      control: "boolean",
+      defaultValue: false,
+    },
+  },
+  decorators: [MuiThemeDecorator],
+};
+
+interface Data {
```

Comment:
This gigantic pile of JS is required to get `<Table>` behaving in the desired way. 

The code is taken from MUI's usage example, modified to meet our desired behaviors while still adhering to their suggestions.

Once wrapped, this functionality should be baked into the component and no longer required in this story.